generators/app/index.js

```diff
@@ -1,11 +1,14 @@
 'use strict';
+
 var Generator = require('yeoman-generator');
-var myPrompts = require('./prompts.js');
 var chalk = require('chalk');
-var yosay = require('yosay');
-var path = require('path');
 var exec = require('child_process').execSync;
+var fs = require('fs-extra');
+var path = require('path');
```

Comment:
If we're cool with `const` down below (node 6+) then we're cool with `const` up here.